web/src/engine/osk/src/lengthStyle.ts

```diff
@@ -12,6 +12,24 @@ export class ParsedLengthStyle implements LengthStyle {
   public constructor(style: LengthStyle | string) {
     let parsed: LengthStyle = (typeof style == 'string') ? ParsedLengthStyle.parseLengthStyle(style) : style;
 
+    const fallback: LengthStyle = {
+      val: 1,
```

Comment:
Maybe a comment would be helpful?
```suggestion
      val: 1, // 100%
```